src/config.ts

```diff
@@ -0,0 +1,14 @@
+require('dotenv').config();
+
+export const mySQLConfig = {
```

Comment:
let's define Typescript Config type here.
@svirgunvolodia can help you as he is working with TS

Reply:
@kanoru3101 Here is an [example](https://github.com/WiseEngineering/crispy-cli/blob/master/src/config.ts) of how @melnychukvitaliy make config on cli

Reply:
We can use the file system config instead of `process.env` like on cli. Typescript has some problems with env types so we can avoid some extra checks